pkg/pipelines/service_rolebinding.go

```diff
@@ -0,0 +1,45 @@
+package pipelines
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1rbac "k8s.io/api/rbac/v1"
+)
+
+// createServiceAccount creates a ServiceAccount given saName and secretName
+func createServiceAccount(saName, secretName string) corev1.ServiceAccount {
+	return corev1.ServiceAccount{
+		TypeMeta:   createTypeMeta("ServiceAccount", "v1"),
+		ObjectMeta: createObjectMeta(saName),
+		Secrets: []corev1.ObjectReference{
+			corev1.ObjectReference{Name: secretName},
+		},
+	}
+}
+
+// createRoleBinding creates a RoleBinding given bindingName, saName, roleKind, and roleName
+func createRoleBinding(bindingName, saName, roleKind, roleName string) v1rbac.RoleBinding {
```

Comment:
I wonder if instead of passing in the roleName, we could just pass in a `Role` and pull the Name and Kind from that?

Same for `saName` ?